src/renderer/components/settings/menu.vue

```diff
@@ -1,15 +1,20 @@
 <template>
-    <div id="settings-menu" class="top-bar secondary">
-        <ul class="menu">
-            <router-link v-for="module in Object.keys(modules)" tag="li" active-class="is-active" :to="{ name: 'settings-module', params: { module } }"
-                :key="module">
-                <a>{{ modules[module] }}</a>
-            </router-link>
-            <router-link tag="li" active-class="is-active" :to="{ name: 'settings-advanced' }"
-                key="settings">
-                <a>advanced</a>
-            </router-link>
-        </ul>
+    <div id="settings-menu" class="ui large secondary menu">
```

Comment:
Why is this `large`? and why `secondary`? I think what would best fit here is a `compact menu`.